target/linux/ramips/image/lzma-loader/src/board.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Arch specific code for ramips based boards
+ *
+ * Copyright (C) 2013 John Crispin <blogic@openwrt.org>
+ * Copyright (C) 2018 Tobias Schramm <tobleminer@gmail.com>
+ * Copyright (C) 2023 Antonio Vázquez <antoniovazquezblanco@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation.
+ */
+
+#include <stdint.h>
+
+#if defined(SOC_MT7620) || defined(SOC_RT3883)
+#define UART_BASE			0xb0000c00
+#define UART_THR			(UART_BASE + 0x0004)
+#define UART_LSR			(UART_BASE + 0x001c)
+#define UART_LSR_THRE_MASK	(1 << 6)
```

Comment:
now getting very picky, can we get rid of the leading zeros and bitwise shift like in the defines for other SOC

`1 << 6` is `0x40`

Reply:
Done!